screen-wake-lock/wakelock-request-denied.https.html

```diff
@@ -1,17 +0,0 @@
-<!DOCTYPE html>
-<html>
-<body>
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
-<script src="/resources/testdriver.js"></script>
-<script src="/resources/testdriver-vendor.js"></script>
-<script>
-'use strict';
-
-promise_test(async t => {
-  await test_driver.set_permission({name: 'screen-wake-lock'}, 'denied');
```

Comment:
Do we have a test that checks that the promise is rejected when request() is called without user interaction?

Reply:
Will make sure we have one... or add one now.